src/dialog-service.js

```diff
@@ -82,6 +53,68 @@ export class DialogService {
       return result;
     });
   }
+  
+  openc(settings?: Object): Promise<DialogController> {
```

Comment:
Not sure I was thinking that but it doesn't seem to properly describe what is being done, perhaps `openSynchronously`